respa_admin/templates/respa_admin/forms/_select.html

```diff
@@ -1,7 +1,11 @@
 <div class="form-group {% if not inline %}select-with-label{% endif %}">
     {% if not skip_label %}
         <label class="control-label input-label">
-            {{ field.label }}{% if field.field.required %}*{% endif %}
+            {% if custom_label %}
+                {{ custom_label }}{% if field.field.required %}*{% endif %}
+            {% else %}
+                {{ field.label }}{% if field.field.required %}*{% endif %}
+            {% endif %}
```

Comment:
To prevent declearing `{% if field.field.required %}*{% endif %}` twice these lines could be refactored to 

```

{% if custom_label %} {{ custom_label }} {% else %} {{ field.label }} {% endif %}

{% if field.field.required %}*{% endif %}


```